Bindings/Java/swig/java_common.i

```diff
@@ -220,6 +233,17 @@ SWIG_JAVABODY_PROXY(public, public, SWIGTYPE)
   }
 %}
 
+%extend OpenSim::Object {
+	static OpenSim::Array<std::string> getFunctionClassNames() {
+		  OpenSim::Array<std::string> availableClassNames;
+		  ArrayPtrs<OpenSim::Function> rArray;
+		  Object::getRegisteredObjectsOfGivenType<OpenSim::Function>(rArray);
+		  for (int i=0;i<rArray.size(); i++)
+			availableClassNames.append(rArray[i]->getConcreteClassName());
+		  
+		  return availableClassNames;
+	}
+}
```

Comment:
Nope, I think I added it earlier but was on a different machine so diffing showed it only once. Thanks for the find @chrisdembia